doc/docs/usage/tutorial-vehicle-signals.md

```diff
@@ -148,8 +148,8 @@ It should print:
 
 ```text
 NAME           WORKLOADS      CPU USAGE      FREE MEMORY
-agent_A        2              42.42%         42B
-infotainment   0              42.42%         42B
+agent_A        2              42%         42B
+infotainment   0              42%         42B
```

Comment:
Otherwise the indentation is wrong in the table:
```suggestion
agent_A        2              42%            42B
infotainment   0              42%            42B
```